support/cas-server-support-oidc-core-api/src/main/java/org/apereo/cas/oidc/token/OidcIdTokenGeneratorService.java

```diff
@@ -42,8 +43,14 @@
 @Getter
 public class OidcIdTokenGeneratorService extends BaseIdTokenGeneratorService {
 
-    public OidcIdTokenGeneratorService(final OAuth20ConfigurationContext configurationContext) {
+    /**
+     * The oauth2 scope to attributes filter.
+     */
+    private final OAuth20ProfileScopeToAttributesFilter scopeToAttributesFilter;
```

Comment:
This will be to become part of the configuration context, if it's not already.

Reply:
This was already available in the configuration context. Fixed changes to use it from context instead of setting it in constructor.